packages/plugin-inline/src/Format.ts

```diff
@@ -2,16 +2,31 @@ import { Modifier } from '../../core/src/Modifier';
 import { Modifiers } from '../../core/src/Modifiers';
 import { Attributes } from '../../plugin-xml/src/Attributes';
 
+/**
+ * A Format might represent DOM inline nodes (a, b, strong) that were parsed or
+ * the DOM inline node (a, b, strong) that we might render.
```

Comment:
There is a TODO just below: "remove this reference to DOM."
I would try and rewrite this to not invoke the DOM.
Plus, all the uses of "might" make this very hesitant ^^

Now what this does however is reveal how a `Format` is not fundamentally different from any other `Modifier` apart from its DOM rendering... @dmo-odoo Any input?